### PR TITLE
build executable in docker container

### PR DESCRIPTION
**Pull request by soenkehahn**
Currently this doesn't build mac or windows executable yet, but just moves the compilation of the linux executable into docker.

### CODE REVIEW

**Comment by soenkehahn** [comment]
cc: @dminnear

File: build/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,5 @@
+FROM golang:1.8.5-jessie
+RUN apt-get update && apt-get install --yes upx-ucl
```
Comment (dminnear) [inline build/Dockerfile]: You may wanna add `--yes / -y` to the update command as well

**Review by dminnear — APPROVED** [review]
LGTM

**Comment by soenkehahn** [comment]
@dminnear: I added another commit that pipes the compiled file from the container into the executable. To have it be owned by the current user.

I'm not very happy with the file names `buildWithDocker.sh` and `build/buildWithGo.sh`. Do you have better ideas?

**Pull request merged**